### PR TITLE
Height of code editor increased

### LINKED ISSUE #56: IDE should have more height, more separation from Footer.
When the page open only IDE should take maximum height. Footer shouldn't be visible, unless page is scrolled down.

ide_issue

### PR DESCRIPTION
**Pull request by realslimshanky**
this fixes #56

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -146,7 +146,7 @@ <h2 class="vnu">Online IDE</h2>
 
     <div class="row">
 
-        <div class="col-md-8 colw90">
+        <div class="col-md-8 colw90" id="code_editor">
```
Comment (aayusharora) [inline index.html]: @realslimshanky Why do we need to change the style with id here, Is colw90 used anywhere else in the code ?

**Comment by aayusharora** [comment]
@realslimshanky Can you please check the conflicts.

**Comment by omerjerk** [comment]
@realslimshanky does this need to be merged? :)

File: index.html (hunk 1 above)
Comment (realslimshanky) [inline index.html]: Yes. colw90 is used multiple times and hence made a class.
I just needed that specific block to my react on css hence added an id.

Comment (aayusharora) [inline index.html]: Oh ok @realslimshanky, please check the conflicts.

Comment (realslimshanky) [inline index.html]: @aayusharora resolved.

**Pull request merged**